docs/src/development/transfer-dedicated.md

```diff
@@ -1,149 +1,135 @@
 ---
-title: "Transferring data to and from a Dedicated cluster"
+title: "Transfer data to and from a Dedicated cluster"
 weight: 13
-sidebarTitle: "Syncing to Dedicated"
+sidebarTitle: "Sync to Dedicated"
 ---
 
-## Backing up staging and production files
+Transferring data to and from a Dedicated cluster slightly differs from the grid.
+For advanced command line utilities, you might need granular SSH access to a specific host.
 
-Platform.sh automatically creates a backup of the staging and production instances on a Dedicated cluster every six hours.
-However, those are only useful for a full restore of the environment and can only be done by the Platform.sh team.
-At times, you'll want to make a manual backup yourself.
+## Back up your files
 
-To create a manual ad-hoc backup of all files on the staging or production environment, use the standard `rsync` command.
+Platform.sh automatically creates backups of the Staging and Production environments of a Dedicated cluster every six hours.
+These are only useful to fully restore an environment and are managed by the support team.
+
+You can make a manual, local, backup yourself by downloading data from your environment to your local system with:
 
 ```bash
-rsync -avzP <USERNAME>@<CLUSTER_NAME>.ent.platform.sh:pub/static/ pub/static/
+platform scp --project=<PROJECT_ID> --environment=<ENVIRONMENT> -r remote:<DIRECTORY_TO_SYNCHRONIZE> <LOCAL_DIRECTORY>
```

Comment:
:pick: This would be one of very few places where we use the `=` in the commands. It's just a little surprising to see it. It also seems more confusing with the `remote:`, but that might be just me.

It's also below